src/applications/disability-benefits/2346/constants/index.js

```diff
@@ -6,71 +11,72 @@ export const FETCH_REORDER_BATTERY_AND_ACCESSORIES_INFORMATION =
   'disability-benefits/2346/FETCH_REORDER_BATTERY_AND_ACCESSORIES_INFORMATION';
 export const FETCH_REORDER_BATTERY_AND_ACCESSORIES_INFORMATION_FAILURE =
   'disability-benefits/2346/FETCH_REORDER_BATTERY_AND_ACCESSORIES_INFORMATION_FAILURE';
-
 export const PERM_ADDRESS_SELECTED_SUCCESSFUL =
   'disability-benefits/2346/PERM_ADDRESS_SELECTED_SUCCESSFUL';
-
 export const PERM_ADDRESS_SELECTED_FAILURE =
   'disability-benefits/2346/PERM_ADDRESS_SELECTED_FAILURE';
-
 export const TEMP_ADDRESS_SELECTED_SUCCESSFUL =
   'disability-benefits/2346/TEMP_ADDRESS_SELECTED_SUCCESSFUL';
-
 export const TEMP_ADDRESS_SELECTED_FAILURE =
   'disability-benefits/2346/TEMP_ADDRESS_SELECTED_FAILURE';
-
 export const PERM_ADDRESS_MILITARY_BASE_SELECTED =
   'disability-benefits/2346/PERM_ADDRESS_MILITARY_BASE_SELECTED';
-
 export const PERM_ADDRESS_MILITARY_BASE_DESELECTED =
   'disability-benefits/2346/PERM_ADDRESS_MILITARY_BASE_DESELECTED';
-
 export const PERM_ADDRESS_MILITARY_BASE_SELECTION_FAILURE =
   'disability-benefits/2346/PERM_ADDRESS_MILITARY_BASE_FAILURE';
-
 export const TEMP_ADDRESS_MILITARY_BASE_SELECTED =
   'disability-benefits/2346/TEMP_ADDRESS_MILITARY_BASE_SELECTED';
-
 export const TEMP_ADDRESS_MILITARY_BASE_DESELECTED =
   'disability-benefits/2346/TEMP_ADDRESS_MILITARY_BASE_DESELECTED';
-
 export const TEMP_ADDRESS_MILITARY_BASE_SELECTION_FAILURE =
   'disability-benefits/2346/TEMP_ADDRESS_MILITARY_BASE_FAILURE';
 
 export const schemaFields = {
+  fullName: 'vetFullName',
   permAddressField: 'permanentAddress',
   tempAddressField: 'temporaryAddress',
   emailField: 'email',
   suppliesField: 'supplies',
-  yesOrNoField: 'yesOrNo',
+  viewAddBatteriesField: 'view:AddBatteries',
+  viewAddAccessoriesField: 'view:AddAccessories',
+  currentAddressField: 'currentAddress',
+  newAddressField: 'newAddress',
+  selectedAddressField: 'selectedAddress',
 };
 
-export const MILITARY_STATE_LABELS = [
-  'Armed Forces Americas (AA)',
-  'Armed Forces Europe (AE)',
-  'Armed Forces Pacific (AP)',
-];
+export const militaryStates = states.USA.filter(
```

Comment:
So we actually have a PR that's trying to consolidate all of the state fields across the VA! https://github.com/department-of-veterans-affairs/vets-website/pull/12178

I'll tag this PR in that PR! 🙂

Reply:
@kelsonic yeah! I think that work will make some changes to this file once merged

Reply:
Actually I added a comment in that PR that I specifically didn't modify this form's folder to prevent merge conflicts since I know you're actively developing in it. I don't like dealing with merge conflicts :D

Reply:
@Mottie 💯